content/pt-br/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -0,0 +1,389 @@
+---
+title: Migrando de PodSecurityPolicy Para Controlador de Admissão Integrado
+reviewers:
+- tallclair
+- liggitt
+content_type: task
+min-kubernetes-server-version: v1.22
+---
+
+<!-- overview -->
+
+Esta página descreve o processo de migração de `PodSecurityPolicies` para 
+o controlador de admissão `PodSecurity` integrado. Isso pode ser feito de maneira 
+eficaz usando uma combinação de modos `dry-run` (ensaio/simulação), `audit` e
+`warn`, embora isso se torne mais difícil se PSPs mutáveis forem usados.
+
+## {{% heading "prerequisites" %}}
+
+{{% version-check %}}
+
+- Garantir que a funcionalidade `PodSecurity` do [portal de funcionalidades]
+(/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features) 
+esteja habilitada.
+
+Esta página pressupõe que você já esteja familiarizado com os conceitos básicos da
+[Segurança de Admissão de Pod](/docs/concepts/security/pod-security-admission/).
+
+<!-- body -->
+
+## Abordagem global
+
+Existem várias estratégias que você pode usar para migrar de `PodSecurityPolicy` 
+para a Segurança de Admissão de Pod. As etapas a seguir são um possível caminho 
+de migração, com o objetivo de minimizar os dois riscos, de uma interrupção 
+de produção e/ou uma lacuna de segurança.
+
+<!-- Keep section header numbering in sync with this list. -->
+0. Decida se a Segurança de Admissão de Pod é o adequado para o seu caso de uso.
+1. Revise as permissões do namespace
+2. Simplifique e padronize as `PodSecurityPolicies`
+3. Atualize os namespaces
+   1. Identifique um nível apropriado de segurança de Pod
+   2. Verifique o nível de segurança do Pod
+   3. Aplique o nível de segurança do Pod
+   4. Desvie o `PodSecurityPolicy`
+4. Revise os processos de criação do namespace
+5. Desabilite o `PodSecurityPolicy`
+
+## 0. Decida se a Segurança de Admissão de Pod é o correto para você {#is-psa-right-for-you}
+
+Segurança de Admissão de Pod foi projetado para atender às necessidades de segurança mais 
+comuns, e para fornecer um conjunto padrão de níveis de segurança entre clusters. 
+No entanto, é menos flexível do que o `PodSecurityPolicy`. Notavelmente, os seguintes 
+recursos são suportados pela `PodSecurityPolicy` mas não pela Segurança de Admissão do Pod:
+
+- **Definindo restrições de segurança padrão** - Segurança de Admissão de Pod 
+  é um controlador de admissão não mutável, significando que não modificará 
+  os pods antes de validá-los. Se você estava confiando nesse aspecto do PSP, 
+  você precisará modificar suas cargas de trabalho para atender às restrições 
+  de segurança do Pod, ou usar um [Webhook de admissão mutante]
+  (/docs/reference/access-authn-authz/extensible-admission-controllers/)
```

Comment:
This link "Webhook de admissão mutante" is broken. Remove the newline after the brackspaces.